### PR TITLE
Astro Runtime 10 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/runtime-version-lifecycle-policy.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ The following table contains the exact lifecycle for each published version of A
 | [5](runtime-release-notes.md#astro-runtime-500) (LTS) | 2.3             | April 30, 2022     | April 2024              |
 | [6](runtime-release-notes.md#astro-runtime-600) (LTS) | 2.4             | September 19, 2022 | March 2024              |
 | [9](runtime-release-notes.md#astro-runtime-900) (LTS) | 2.7             | August 18, 2023    | January 2025            |
+| [9](runtime-release-notes.md#astro-runtime-1000) | 2.8            |TBD    | TBD            |
```
Comment (jwitz) [inline astro/runtime-version-lifecycle-policy.md:87]:
```suggestion
| [10](runtime-release-notes.md#astro-runtime-1000) | 2.8            |TBD    | TBD            |
```

File: astro/runtime-release-notes.md [hunk 2]
```diff
@@ -24,6 +24,20 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/open-support-request).
 
+## Astro Runtime 10.0.0
+
+- Release date: December [TBD] 2023
+- Airflow version: 2.8.0
+
+### Airflow 2.8
+
+Astro Runtime 9.2.0 includes same-day support for Apache Airflow 2.8, which includes a number of new features and improvements. Most notably, Airflow 2.8 includes the following changes:
```
Comment (ephraimbuddy) [inline astro/runtime-release-notes.md:34]:
```suggestion
Astro Runtime 10.0.0 includes same-day support for Apache Airflow 2.8, which includes a number of new features and improvements. Most notably, Airflow 2.8 includes the following changes:
```

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -24,6 +24,20 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+- The new object storage features make it easier to work with popular object storage systems like S3 and GCS. You can use new  abstract object types to work with files in multiple different object storage systems without writing system-specific code.
+- You can now document your DAGs using HTML to provide richer web-based content for understanding and interacting with DAGs in the Airflow UI. This feature replaces `custom_html_form` attribute.
```
Comment (ephraimbuddy) [inline astro/runtime-release-notes.md:37]:
```suggestion
```
We disabled html form. I think we don't need to mention it here. See :https://github.com/apache/airflow-site/pull/906

**Review by ephraimbuddy — COMMENTED** [review]
We need to document the logging provider fixes

File: astro/runtime-release-notes.md [hunk 4]
```diff
@@ -24,6 +24,30 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+For more information about the major changes in this release, see the [Airflow blog](https://airflow.apache.org/blog/airflow-2.8.0/).
+
+### Additional improvements
+
+- When you export logs to Datadog from Astro, you can now filter the logs in Datadog by log type.
+
+### Bug fixes
+
+- Fixed an issue in Astro where all Airflow task logs exported to Datadog appeared as `INFO` logs regardless of their actual log type.
+- Fixed an issue in Astro where logging features could be disrupted if you set `AZURE_CLIENT_ID` or `AZURE_TENANT_ID` as environment variables.
```
Comment (ephraimbuddy) [inline astro/runtime-release-notes.md:49]: We can remove the AZURE_CLIENT_ID. There's still work that needs to be done in astro side

Comment (jwitz) [inline astro/runtime-release-notes.md:49]:
```suggestion
- Fixed an issue in Astro where logging features could be disrupted if you set `AZURE_TENANT_ID` as environment variables.
```

File: astro/runtime-release-notes.md [hunk 5]
```diff
@@ -24,6 +24,30 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+- The Airflow UI **Grid** view now supports filtering on multiple run states at once so that you can compare runs across states.
+
+For more information about the major changes in this release, see the [Airflow blog](https://airflow.apache.org/blog/airflow-2.8.0/).
+
+### Additional improvements
+
+- When you export logs to Datadog from Astro, you can now filter the logs in Datadog by log type.
+
+### Bug fixes
+
```
Comment (jwitz) [inline astro/runtime-release-notes.md:47]:
```suggestion

- Fixed an issue where you could not set `AIRFLOW__WEBSERVER__NAVBAR_TEXT_COLOR` to customize the color of text in the Airflow UI.
```

File: astro/runtime-release-notes.md [hunk 6]
```diff
@@ -24,6 +24,30 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
 
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://cloud.astronomer.io/open-support-request).
 
+## Astro Runtime 10.0.0
+
+- Release date: December 15, 2023
```
Comment (jwitz) [inline astro/runtime-release-notes.md:29]:
```suggestion
- Release date: December 18, 2023
```

File: astro/runtime-version-lifecycle-policy.md [hunk 7]
```diff
@@ -84,6 +84,8 @@ The following table contains the exact lifecycle for each published version of A
 | [5](runtime-release-notes.md#astro-runtime-500) (LTS) | 2.3             | April 30, 2022     | April 2024              |
 | [6](runtime-release-notes.md#astro-runtime-600) (LTS) | 2.4             | September 19, 2022 | March 2024              |
 | [9](runtime-release-notes.md#astro-runtime-900) (LTS) | 2.7             | August 18, 2023    | January 2025            |
+| [10](runtime-release-notes.md#astro-runtime-1000) | 2.8                  |December 25, 2023    | June 2024            |
```
Comment (jwitz) [inline astro/runtime-version-lifecycle-policy.md:87]:
```suggestion
| [10](runtime-release-notes.md#astro-runtime-1000) | 2.8                  |December 18, 2023    | June 2024            |
```

File: astro/runtime-release-notes.md [hunk 8]
```diff
@@ -24,6 +24,31 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
+
+For more information about the major changes in this release, see the [Airflow blog](https://airflow.apache.org/blog/airflow-2.8.0/).
+
+### Additional improvements
+
+- When you export logs to Datadog from Astro, you can now filter the logs in Datadog by log type.
+
+### Bug fixes
+
+- Fixed an issue where you could not set `AIRFLOW__WEBSERVER__NAVBAR_TEXT_COLOR` to customize the color of text in the Airflow UI.
```
Comment (jwitz) [inline astro/runtime-release-notes.md:48]:
```suggestion
```

**Pull request merged**